src/marshmallow/fields.py

```diff
@@ -465,13 +474,20 @@ class Nested(Field):
     :param unknown: Whether to exclude, include, or raise an error for unknown
         fields in the data. Use `EXCLUDE`, `INCLUDE` or `RAISE`.
     :param kwargs: The same keyword arguments that :class:`Field` receives.
+
+    .. versionchanged:: 3.1.0
+        Deprecated ``only``, ``exclude``, and ``unknown`` parameters.
```

Comment:
It looks like the deprecation warning was reverted, but the documentation still says these parameters are being deprecated. I would be fine with deprecating them.

Reply:
Ah, woops, forgot to remove this. 

See https://github.com/marshmallow-code/marshmallow/pull/1382#issuecomment-530147731 . My thinking is that if we're going to support string names in the short term, we should hold off on deprecating `only` etc. because that's the only way to pass those args when using strings.